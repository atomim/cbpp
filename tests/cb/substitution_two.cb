#define SQUARE(a) (a*a)
Print "5^2 = " + SQUARE(5)
WaitKey
