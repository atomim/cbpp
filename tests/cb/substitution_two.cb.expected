// #define SQUARE(a) (a*a)
Print "5^2 = " + (5*5)
WaitKey
